app/assets/stylesheets/components/_alert.scss

```diff
@@ -16,4 +16,4 @@
 
 .alert-success { background-color: #e7f4e3; }
 .alert-progress { background-color: $blue-light; }
-.alert-danger { background-color: #fdd9d7; }
+.alert-danger { background-color: #ffe7eb; }
```

Comment:
i believe this is the only place we're using them, in which case I prefer no new variables